app/frontend/src/features/user/UserHeader.tsx

```diff
@@ -51,7 +51,7 @@ export default function UserHeader({ children, user }: UserHeaderProps) {
       {user.lastActive && (
         <Box mb={2}>
           <Typography component="p" variant="caption" gutterBottom>
-            {timeAgo(timestamp2Date(user.lastActive))}
+            {user.lastActive ? timeAgo(timestamp2Date(user.lastActive)) : undefined}
```

Comment:
```suggestion
            {user.lastActive && timeAgo(timestamp2Date(user.lastActive))}
```

Reply:
Not sure why I changed this, since we're already handling user.lastActive above. Reverting back